custom/icds_reports/views.py

```diff
@@ -528,12 +528,13 @@ def get(self, request, *args, **kwargs):
                 include_test
             )
         elif step == 'beneficiary':
-            data = get_awc_report_beneficiary(
-                domain,
-                config['awc_id'],
-                tuple(month.timetuple())[:3],
-                tuple(two_before.timetuple())[:3]
-            )
+            if 'awc_id' in config:
```

Comment:
What should happen if awc_id is not in the config?

Reply:
@emord will return a empty data.